### ISSUE TITLE
Make the Request link that is displayed in the console be clickable.

### ISSUE THREAD

**Issue by angamardhana** [a85/POSTMan-Chrome-Extension#4682]
Any time I test a script, the requested url is printed out on the console. 
It is frustrating to have to cut and paste it to a new browser window. Even double clicking it to copy doesn't work. I have to move the mouse over carefully over the whole url to copy.

Can you have the whole link hyperlinked. That would be great ! 

Thanks.

- jay 
![console](https://user-images.githubusercontent.com/40009667/41321584-6f765d74-6e59-11e8-91a7-bfa669bc74c3.png)

**Comment by StrohmyerTR** [comment]
I would like for this to apply to _any_ console log statements, so I can dynamically build a clickable URL which points to my team's error logging application.

**Comment by azgo** [comment]
Absolutely, that would be so helpful! Please parse them to make them clickable.
